organization/api.go

```diff
@@ -19,6 +19,12 @@ func Get(ctx context.Context, idOrSlug string) (*clerk.Organization, error) {
 	return getClient().Get(ctx, idOrSlug)
 }
 
+// GetWithParams retrieves details for an organization.
+// The organization can be fetched by either the ID or its slug.
+func GetWithParams(ctx context.Context, idOrSlug string, params *GetParams) (*clerk.Organization, error) {
```

Comment:
❓ Seems like unrelated changes to the PR description

Reply:
I ran `go generate ./...` so this is probably was forgotten to run in a previous PR